src/org/loklak/api/search/SearchServlet.java

```diff
@@ -269,16 +272,16 @@ public void run() {
                 post.recordEvent("backend_time", System.currentTimeMillis() - start);
 
             }
-    
+
             // check the latest user_ids
             DAO.announceNewUserId(tl);
-            
```

Comment:
Please revert this line, it's not part of the issue. If you want to do some house-keeping or cleaning, please create a new PR and Issue for it.